Logger/BookRecord.cs

```diff
@@ -0,0 +1,17 @@
+﻿
+namespace Logger;
+
+public record class BookRecord(string ClassName) : BaseEntity
+{
+    
+    // We implemented the Name property implicitly because it is an is-a relationship 
+    public override string Name 
+    {
+        get 
+        {
+            return $"{ClassName}";
+        }
```

Comment:
```suggestion
        get 
        {
            return ClassName;
        }
```
Why does this need a interpolated string?